showcase/src/app/demo-ui/pages/progress-indicator/demo-progress-indicator-page.component.scss

```diff
@@ -0,0 +1,8 @@
+.progress-indicator-example {
+  text-align: center;
+}
+
+.progress-indicator-example span {
+  border: 1px solid color(mat-color($grey-palette, 400));
```

Comment:
If you are using material colors the this file should be renamed to `demo-progress-indicator-page-theme.scss` as we do with other theme styling and import it in `src/styles/_theme.scss`.

Or... you can simply use a fixed color instead of a paletter color and leave the file as it is ;)